src/main/resources/messages.properties

```diff
@@ -1,27 +1,27 @@
-typeMismatch.courseConfigurationForm.totalClassMinutes = Must be a valid number.
-Min.courseConfigurationForm.totalClassMinutes = Must be greater than 0.
+typeMismatch.courseConfigurationForm.totalClassMinutes=Must be a valid number.
+Min.courseConfigurationForm.totalClassMinutes=Must be greater than 0.
 
-typeMismatch.courseConfigurationForm.defaultMinutesPerSession = Must be a valid number.
-Min.courseConfigurationForm.defaultMinutesPerSession = Must be greater than 0.
+typeMismatch.courseConfigurationForm.defaultMinutesPerSession=Must be a valid number.
+Min.courseConfigurationForm.defaultMinutesPerSession=Must be greater than 0.
 
-typeMismatch.rosterForm.sectionModels.attendances.minutesMissed = Must be a valid number.
-Min.rosterForm.sectionModels.attendances.minutesMissed = Must be greater than 0.
-SelectivelyRequired.rosterForm.sectionModels.attendances.minutesMissed = Required when marked as Tardy.
+typeMismatch.rosterForm.sectionModels.attendances.minutesMissed=Must be a valid number.
+Min.rosterForm.sectionModels.attendances.minutesMissed=Must be greater than 0.
+SelectivelyRequired.rosterForm.sectionModels.attendances.minutesMissed=Required when marked as Tardy.
 
-ExceedTotal.courseConfigurationForm.defaultMinutesPerSession = Must not exceed total class minutes.
+ExceedTotal.courseConfigurationForm.defaultMinutesPerSession=Must not exceed total class minutes.
 
-typeMismatch.makeupForm.entries.minutesMadeUp = Must be a valid number.
-Min.makeupForm.entries.minutesMadeUp = Must be greater than 0.
+typeMismatch.makeupForm.entries.minutesMadeUp=Must be a valid number.
+Min.makeupForm.entries.minutesMadeUp=Must be greater than 0.
 
-Required.courseConfigurationForm.entries.assignmentName = Assignment Name is required.
-Required.courseConfigurationForm.entries.assignmentPoints = Assignment Points is a required field and has to be greater than 0.
-Required.courseConfigurationForm.entries.statusPoints = All status point fields are required.
-Max.courseConfigurationForm.entries = Point values must be set between 0 and 100.
+Required.courseConfigurationForm.entries.assignmentName=Assignment Name is required.
+Required.courseConfigurationForm.entries.assignmentPoints=Assignment Points is a required field and has to be greater than 0.
+Required.courseConfigurationForm.entries.statusPoints=All status point fields are required.
+Max.courseConfigurationForm.entries=Point values must be set between 0 and 100.
 
-typeMismatch.makeupForm.entries.dateMadeUp = Must be a valid Date dd/mm/yyyy
+typeMismatch.makeupForm.entries.dateMadeUp=Must be a valid Date dd/mm/yyyy
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge keys "typeMismatch.makeupForm.entries.dateMadeUp, typeMismatch.makeupForm.entries.dateOfClass" that have the same value "Must be a valid Date dd/mm/yyyy". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aduplicated-values)